src/integrator/LangevinThermostatOnRadius.py

```diff
@@ -0,0 +1,77 @@
+#  Copyright (C) 2017
+#      Max Planck Institute for Polymer Research
+#  
+#  This file is part of ESPResSo++.
+#  
+#  ESPResSo++ is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  
+#  ESPResSo++ is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+
+
+r"""
+********************************************
+**espressopp.integrator.LangevinThermostatOnRadius**
+********************************************
+Langevin Thermostat for Radii of Particles
```

Comment:
I don't have any reference for this. Actually, this is just normal langevin thermostat that only affect not the position but the radius of particles.
For confirmation, `LangevinThermostat` only affect the position of particles.